source/jormungandr/jormungandr/scenarios/default.py

```diff
@@ -130,9 +130,10 @@ def parse_journey_request(self, requested_type, request):
             req.journeys.streetnetwork_params.origin_mode = "bss"
         if req.journeys.streetnetwork_params.destination_mode == "bike_rental":
             req.journeys.streetnetwork_params.destination_mode = "bss"
-        if "forbidden_uris[]" in request and request["forbidden_uris[]"]:
-            for forbidden_uri in request["forbidden_uris[]"]:
-                req.journeys.forbidden_uris.append(forbidden_uri)
+        for forbidden_uri in get_or_default(request, "forbidden_uris[]", []):
```

Comment:
I'm not sure what `get_or_default` do, but I think you can just do 
`for forbidden_uri in request.get("forbidden_uris[]", []):`